include/nlohmann/detail/conversions/from_json.hpp

```diff
@@ -464,7 +464,8 @@ void from_json(const BasicJsonType& j, std_fs::path& p)
 {
     if (JSON_HEDLEY_UNLIKELY(!j.is_string()))
     {
-        JSON_THROW(type_error::create(302, "type must be string, but is " + std::string(j.type_name()), j));
+        // Not tested because of #3377 (related #3070)
```

Comment:
I assume the test will be deleted with #3377, right?

Reply:
I was just triggered by the coverage exclusion - we should be able to have a test for every part of the library - especially for "visible" effects like exceptions. I would like to keep it as is here and address it in a later issue.

Reply:
Alright. We have issue #3377 to keep track of it.